config/routes.rb

```diff
@@ -0,0 +1,77 @@
+Rails.application.routes.draw do
+  # The priority is based upon order of creation: first created -> highest priority.
+  # See how all your routes lay out with "rake routes".
+
+  # You can have the root of your site routed with "root"
+  # root 'welcome#index'
+
+  # Example of regular route:
+  #   get 'products/:id' => 'catalog#view'
+
+  # set root routing
+  root 'tasks#index'
+
+  # set Task routes
```

Comment:
If you know you're going to pretty much use all of the HTTP verbs for a given controller, you can just use handy dandy
   resources :tasks
if you haven't already, read over the commented out code that rails automatically generates in this file, it includes a lot of useful info on potential ways to name routes. (also there's nothing wrong with the way you did it, just letting you know the magic of resources :whateveryourmodelnameis)